### PR TITLE
Upgrade xterm to v5.5.0

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description

Makes sure we follow the renaming of the package. It _could_ fix bugs as well, ya know :).

See https://github.com/xtermjs/xterm.js/releases/tag/5.4.0 about moving to `@xterm/` for info about the package renaming.

It also does some cleanup of our `package.json` like moving `@types` packages to `devDependencies`.

https://ft-bump-xterm.preview.gitpod-dev.com/prebuilds

- [x] /werft with-preview

### CODE REVIEW

**Comment by geropl** [comment]
Checked the version updates, incl. the yarn.lock (which looked daunting at first): the change LGTM, the yarn.lock changes seem all be updates, and mostly radix-ui related. :heavy_check_mark: 

Don't have time to test today, so somebody else has to do that. :+1:

**Comment by filiptronicek** [comment]
Streaming logs continues to work as expected

image

File: yarn.lock
Comment (filiptronicek) [inline yarn.lock:1]: Used `yarn upgrade-interactive` for this one, updating mostly radix-ui components that make up components in Podkit.

**Review by geropl — APPROVED** [review]
Code LGTM, happy to merge if you are confident in your tests.

Otherwise, please ask somebody else to help out :pray:

**Comment by mustard-mh** [comment]
I were testing with browser terminal yesterday actually 🤣 Then recognized it's the streaming log view on dashboard before the last second approval with LGTM.

**Pull request merged**
